tests/rkt_tests.go

```diff
@@ -312,6 +312,31 @@ func runRktAsUidGidAndCheckOutput(t *testing.T, rktCmd, expectedLine string, exp
 	}
 }
 
+func runRkt(t *testing.T, rktCmd string, uid, gid int) (string, int) {
+	child, err := gexpect.Command(rktCmd)
+	if err != nil {
+		t.Fatalf("cannot exec rkt: %v", err)
+	}
+	if gid != 0 {
```

Comment:
Sorry if I was not clear enough. I was surprised by the the `gid!=0` conditional which also control whether the uid will be taken into account or not. Looking at the function signature, I was expecting it to always set uid/gid.